patches/sitecustomize.py

```diff
@@ -101,7 +101,7 @@ def new_configure(*args, **kwargs):
             client_options = kwargs['client_options']
         else:
             client_options = {}
-        client_options['api_endpoint'] = os.environ['KAGGLE_DATA_PROXY_URL']
+        client_options['api_endpoint'] = os.environ['KAGGLE_GRPC_DATA_PROXY_URL']
```

Comment:
If transport is REST, wouldn't we want to use the old `KAGGLE_DATA_PROXY_URL`?

Reply:
Very good catch, thanks!